sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -186,39 +186,34 @@ export interface ExtractKeyPhrasesSuccessResult extends TextAnalyticsSuccessResu
 export interface HealthcareEntitiesArray extends Array<HealthcareResult> {
 }
 
-// @public (undocumented)
+// @public
 export type HealthcareEntity = Entity & {
```

Comment:
```suggestion
export interface HealthcareEntity extends Entity {
```

This generates better documentation than `type` if it will work.

Reply:
Fixed